pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -1946,8 +1946,11 @@ default 'raise'
         >>> pd.NaT.tz_localize()
         NaT
         """
-        if ambiguous == "infer":
-            raise ValueError("Cannot infer offset with only one time.")
+        if not isinstance(ambiguous, bool) and ambiguous not in {"NaT", "raise"}:
+            raise ValueError(
+                        "ambiguous parameter must be one of: "
```

Comment:
let's quote the parameter name, else it'll be ambiguous (😄 )
```
                        "'ambiguous' parameter must be one of: "
```

Reply:
done :)